src/app/config/hypeEvmStrategies.ts

```diff
@@ -253,4 +255,88 @@ export const HYPE_EVM_CONFIG: Strategy[] = [
     ],
     externalAppURL: "https://app.hyperbeat.org/staking/behype",
   },
+  {
+    strategy: "Hyperbeat Borrow: hbHYPE/WHYPE",
+    start: "Aug-07-2025 02:26:58 AM UTC",
+    owner: "0xC90780cB9aab4dc0781E2E7b5197846C2907F82B",
+    fixedValue: { value: 0.5549, asset: "HYPE" }, // total collateral
+    points: [
+      {
+        type: POINTS_ID_HYPERBEAT_S1,
+      },
+      {
+        type: POINTS_ID_KINETIQ_S1,
+      },
+      // {
+      //   type: POINTS_ID_HYPERLEND_S1,
+      //   expectedPointsPerDay: { value: 1, baseAsset: "USD" },
+      // },
+      // {
+      //   type: POINTS_ID_HYPURRFI_S1,
+      //   expectedPointsPerDay: { value: 1, baseAsset: "USD" },
+      // },
+      // {
+      //   type: POINTS_ID_HYPERSWAP_S1,
+      //   expectedPointsPerDay: { value: 1, baseAsset: "USD" },
+      // },
+      // {
+      //   type: POINTS_ID_TIMESWAP_S1,
+      //   expectedPointsPerDay: { value: 1, baseAsset: "USD" },
+      // },
+      // {
+      //   type: POINTS_ID_SILHOUETTE_S1,
+      //   expectedPointsPerDay: { value: 1, baseAsset: "USD" },
+      // },
+      { type: POINTS_ID_UPSHIFT_S2 },
+    ],
+    externalAppURL:
+      "https://app.hyperbeat.org/borrow/0x19e47d37453628ebf0fd18766ce6fee1b08ea46752a5da83ca0bfecb270d07e8",
+  },
+  {
+    strategy: "Kinetiq Earn: vkHYPE",
+    start: "Aug-07-2025 02:37:40 AM UTC",
+    owner: "0xD857a632b829611Dc7397F87A7c6BA6821eF5A03",
+    fixedValue: { value: 0.2996, asset: "HYPE" },
+    points: [{ type: POINTS_ID_KINETIQ_S1 }],
+    externalAppURL: "https://kinetiq.xyz/earn/kinetiq-earn",
+  },
+  {
+    strategy: "Pendle: Hold YT-beHYPE-29OCT2025",
```

Comment:
Maybe can do a format more like `Pendle: Hold beHYPE YTs (29 OCT 2025)` to match what the FE will likely show